style.css

```diff
@@ -0,0 +1,147 @@
+@import url('https://fonts.googleapis.com/css2?family=Josefin+Sans:wght@100&family=Roboto+Slab&display=swap');
+
+html {
+    font-family: 'Josefin Sans', sans-serif;
+    color: #143914;
+    font-size: 12px;
+}
+
+*, *::before, *::after {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    background-color: white;
+}
+
+main {
+    background-color: #d9f2e6;
+    text-align: center;
+    padding-top: 50px;
+}
+
+a {
+    color: #669900;
+    font-family: 'Josefin Sans', sans-serif;
+}
+
+a:hover {
+    color: #e00dbd
+}
+
+a:focus{
+    color:#e00dbd;
+}
+
+p {
+    text-align: center;
+    padding-top: 15px;
+    padding-bottom: 20px;
+    padding-left: 0px
+    padding-right: 0px;
+    font-size: 2em;
+    font-weight: bold;
+    color: #434323;
+}
+
+h1, h2 {
+    color: #143914;
+    text-align: center;
+    font-weight: normal;
+}
+
+h1 {
+    font-size: 5rem;
+}
+
+.nav_bar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-color: white;
+    height: 30px;
+    padding: 30px;
+    padding-bottom: 20px;
+    width: 100%;
+}
+
+.welcome {
+    font-size: 2em;
+    font-weight: bold;
+    padding-top: 0px;
+}
+
+.links_nav {
+    font-size: 2em;
+}
+
+.plants_collection {
+    background-color: #d9f2e6;
+    font-size: 1rem;
+    font-family: 'Roboto Slab', serif;
+    text-align: center;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    padding-left: 50px;
+    padding-right: 50px;
+    padding-bottom: 50px;
+    padding-top: 50px;
+}
+
+.plants {
+    width: 280px;
+    background-color: white;
```

Comment:
keep the values of the properties the same -- you've used hex codes for colours before so even though white is a simple colour, use #FFFFFF for white!

Reply:
changed